### DISCUSSION TITLE
Setting up Hermes Relayer between localChain and BandChain testnet

### DISCUSSION THREAD

**Discussion by AntoninPS** [cosmos/community#42] [Chain Builders (Cosmos SDK)] [answered]
Hello guys,

I've successfully set up a chain and an IBC enabled module using the Ignite CLI. My objective is to initiate Oracle requests from my module to the BandChain testnet (band-laozi-testnet6). To facilitate communication between the two chains, I've established a configuration involving a Hermes Relayer. I followed the steps outlined in this guide: Hermes Relayer Setup.

Upon completing this configuration process, I verified the establishment of a channel between the chains through two methods. Firstly, by executing the command:

`hermes query channels --show-counterparty --chain myChain`

Alternatively, I observed the channel's presence by visiting the BandChain testnet explorer at: BandChain Testnet Explorer.

However, I've encountered an issue when attempting to observe the packets I'm sending from my module to the BandChain. Specifically, I'm working on transmitting data from my module using the igniteCLI tool. I've implemented a msg_server_oracle_request_packet_data.go file that should be responsible for sending the required packet to the BandChain. Upon executing the CLI command, I receive a transaction hash. Although inspecting the transaction details reveals no errors, I've noticed that I'm not receiving any response from the BandChain. Furthermore, this transaction isn't visible on the BandChain testnet explorer.

I'm open to suggestions or insights regarding why this setup might not be functioning as expected.

Happy to share more information about my configuration if needed!

Thank you.

**Comment by seanchen1991** [comment]
@AntoninPS I've opened your query as an [issue](https://github.com/informalsystems/hermes/issues/3570) on the Hermes repo. Let's continue the discussion there 🙂

**Comment by seanchen1991** [comment, marked as answer]
Summarizing how this issue was resolved:

@AntoninPS's Hermes instance was returning a `"packet commitment bytes are not equal"` error message. This is usually an indication that the timeout packet sent to the chain is malformed. If the problem was indeed a Hermes issue, this would likely not be an isolated instance. The most likely answer is that the chain itself was misconfigured. 

@AntoninPS's response on how the issue was resolved:
> The issue was not with the relayer configuration itself, but how we encoded our ibcPacket, which was incompatible with the expected packets on the receiving chain (bandchain).
